stats.tex

```diff
@@ -0,0 +1,57 @@
+%% Stats
+The \verb|sympy.stats| module provides random variable types and methods for
+determination of statistical properties of expressions involving random
+variables. The variables can be either continuous or discrete, the latter being
```

Comment:
OK, I'll change it to:
```
The \verb|sympy.stats| module provides random variable types and methods for
determination of statistical properties of expressions involving random
variables, which can be either continuous or discrete, the latter ones being
further divided into finite and infinite. The variables are associated
with probability densities on corresponding domains and internally defined
in terms of probability spaces. 
```
(changing also "They" to "The variables" in the second sentence, because it wouldn't be clear what "They" refers to)